app/server/utils/blockchain_tasks.py

```diff
@@ -178,7 +190,7 @@ def make_token_transfer(self, signing_address, token,
         :param token: ERC20 token being transferred
         :param from_address: address of wallet sending token
         :param to_address:
-        :param amount: the NON WEI amount being sent, eg 2.3 Dai
+        :param amount: the CENTS amount being sent, eg 2300 Cents = 2.3 Dollars
         :param dependent_on_tasks: list of task IDs that must complete before txn will attempt
         :return: task id for the transfer
```

Comment:
shouldn't it be `:return: task uuid for the transfer`?

see 
https://github.com/teamsempo/SempoBlockchain/pull/88/files#diff-1930434c0329c8a9ba0046d6053c39a2R269